addons/account_facturx/models/account_invoice.py

```diff
@@ -295,8 +295,8 @@ def _get_attachment_content(attachment):
                         # '[::2]' because it's a list [fn_1, content_1, fn_2, content_2, ..., fn_n, content_2]
                         for filename_obj, content_obj in list(pycompat.izip(embedded_files, embedded_files[1:]))[::2]:
                             content = content_obj.getObject()['/EF']['/F'].getData()
-
-                            if filename_obj == 'factur-x.xml':
+                            # FROM 14.0 : filename formating changed (eg.: 'INV_2020_12_0001_facturx.xml')
+                            if filename_obj == 'factur-x.xml' or filename_obj.endswith('facturx.xml'):
```

Comment:
Here is how we check if it is a factur-x document in version 14.0
https://github.com/odoo/odoo/blob/7331b404bff7f803299a3bfddcf0bd3d040f2cd8/addons/account_edi_facturx/models/account_edi_format.py#L119

Wouldn't it be better to do it this way instead of relying on the filename, which can still be different if it comes from another software than Odoo?

Reply:
@william-andre Done :+1: